storage/api/Storage.Samples/MakePublic.cs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// [START storage_set_bucket_public_iam]
+// [START storage_make_public]
```

Comment:
We'll have to do this in two PRs, I'll explain tomorrow.

Reply:
Updated with both tags for now. Will create another PR post this one getting approved